core/src/nucleus/ribosome/api/commit.rs

```diff
@@ -15,7 +15,7 @@ use wasmi::{RuntimeArgs, RuntimeValue, Trap};
 #[derive(Deserialize, Default, Debug, Serialize)]
 struct CommitAppEntryArgs {
     entry_type_name: String,
-    entry_content: String,
+    entry_value: String,
```

Comment:
This is one of those changes that breaks the HDK if the copy of this struct on the HDK side is not changed in the same way!